tests/puppeteer/pages/BO/customers/index.js

```diff
@@ -331,4 +354,38 @@ module.exports = class Customers extends BOBasePage {
     await this.waitForSelectorAndClick(this.saveButton);
     return this.getTextContent(this.alertSuccessBlockParagraph);
   }
+
+  // Export methods
+  /**
+   * Click on link to export customers to a csv file
+   * @return {Promise<void>}
+   */
+  async exportDataToCsv() {
+    await Promise.all([
+      this.page.click(this.customerGridActionsButton),
+      this.page.waitForSelector(`${this.gridActionDropDownMenu}.show`, {visible: true}),
+    ]);
+    await Promise.all([
+      this.page.click(this.gridActionExportLink),
+      this.page.waitForSelector(`${this.gridActionDropDownMenu}.show`, {hidden: true}),
+    ]);
+  }
+
+  /**
+   * Get customer from table in csv format
+   * @param row
+   * @return {Promise<string>}
+   */
+  async getCustomerInCsvFormat(row) {
+    const customer = await this.getCustomerFromTable(row);
+    return `${customer.id};`
+      + `${customer.socialTitle};`
+      + `${customer.firstName};`
+      + `${customer.lastName};`
+      + `${customer.email};;`
```

Comment:
is that empty csv case normal ?

Reply:
This is for column 'Company', it's visible only if B2B mode is enabled.
So since B2B mode is disabled by default, I added the empty case, because the column is added in csv file anyway .

Reply:
May be add a comment ;)

Reply:
Added in function's description

Reply:
Alright